packages/clerk-js/src/core/services/authentication/SessionCookieService.ts

```diff
@@ -64,7 +64,11 @@ export class SessionCookieService {
     try {
       this.updateSessionCookie(await this.getNewToken());
     } catch (e) {
-      return this.handleGetTokenError(e);
+      try {
+        return this.handleGetTokenError(e);
+      } catch {
+        void this.clerk.handleUnauthenticated();
```

Comment:
Can we move the nested catch inside `handleGetTokenError`?

Reply:
done